### ISSUE TITLE
Add logical dimension values for touch-action property, since logical is/has shipped

### ISSUE THREAD

**Issue by jonjohnjohnson** [w3c/pointerevents#272]
https://w3c.github.io/pointerevents/#the-touch-action-css-property
https://drafts.csswg.org/css-logical-1/
https://caniuse.com/#feat=pointer
https://caniuse.com/#feat=css-logical-props

With Webkit being the last major vendor to work on pointer events, initially deployed in STP Release 67, as well as Edge being the only vendor without impending support for logical properties and values, it seems the `touch-action` property needs to be updated to accept a set of logical values.

cc @mbrubeck @atanassov

**Comment by patrickhlauke** [comment]
i assume you mean logical values for `pan-x`/`pan-y` in this case, right?

**Comment by jonjohnjohnson** [comment]
@patrickhlauke yep and any specifics of logical/physical language in the examples.

**Comment by NavidZ** [comment]
I guess this would also include pan-left/right/up/down in the extension doc.

**Comment by patrickhlauke** [comment]
I'll note that this is probably not so much about replacing those values, but complementing them with some form of logical axis/direction type values (that adapt to different writing systems etc). certainly `pan-x`/`pan-y` seem fine to me as they, in my mind at least, relate to the x/y coordinate system (which does not change for different writing systems) (as i see those being more low-level related to coordinates)

**Comment by jonjohnjohnson** [comment]
@patrickhlauke of course I hope it's only adding values, not replacing. No physical property/value has been replaced, only added afaik.

Even if someone does want to use scrolling/panning/interactions in the physical mapping, that's fine, but we also have a bunch of new ways to provide scrolling/panning/interactions that are related to a logical mapping. Think `overflow-inline: scroll`, `scroll-snap-type: block`, even if simply `block`/`inline`.

**Comment by patrickhlauke** [comment]
roughed out a potential addition here https://github.com/w3c/pointerevents/pull/496
